bin/runtest_common.mli

```diff
@@ -2,6 +2,7 @@ open Import
 
 val make_request
   :  contexts:Context.t list
+  -> scontexts:Super_context.t Context_name.Map.t
```

Comment:
Seems redundant to pass both `contexts` and `scontexts`. How about just making `contexts` be your new super context map? Or looking up the super context by context name inside this function.